test-data/run.test

```diff
@@ -4039,3 +4103,46 @@ with assertRaises(TypeError, "missing required keyword-only argument 'x'"):
     kwonly4(y=1)
 with assertRaises(TypeError, "missing required keyword-only argument 'x'"):
     kwonly4()
+
+# varargs tests
+assert varargs1() == ()
```

Comment:
What happens if somebody calls these using caller `*args` or `**kwargs` from Python?

Reply:
It all goes through the same code path